change/@ni-nimble-tokens-7d1352e1-f82c-48b8-94fd-d86131297a6d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

let's do a patch just to exercise the release steps since the scripts changed

```suggestion
  "type": "patch",
```